api-tests/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig.json",
+  "compilerOptions": {
```

Comment:
please add `strict: true` here to enable strict checks

https://www.typescriptlang.org/tsconfig#strict